lib/hours.php

```diff
@@ -6,637 +6,598 @@
  * @since 1.2.1
  */
 
-if (isset($_GET["cleanCache"])) {
-	$clean = $_GET["cleanCache"];
+if ( isset( $_GET['cleanCache'] ) ) {
+	$clean = $_GET['cleanCache'];
 } else {
 	$clean = 0;
 }
 
 $resetCache = 0;
 
-if ($clean == "1") $resetCache = 1;
+if ( $clean == '1' ) { $resetCache = 1; }
 $halfCache = 0;
 
-if (isset($_GET["locReset"]))
-	$locationReset = $_GET["locReset"];
+if ( isset( $_GET['locReset'] ) ) {
+	$locationReset = $_GET['locReset']; }
 
-$expireLong = 60*5; // 24 hours
-$expireShort = 60*5; // 5 minutes
-date_default_timezone_set("America/New_York");
+$expireLong = 60 * 5; // 24 hours
+$expireShort = 60 * 5; // 5 minutes
```

Comment:
I know this isn't related to the goal of this PR but I would like to note that these two maths do not seem to match the comment value :P